tests/test.py

```diff
@@ -0,0 +1,38 @@
+import time
+import logging
+import os
+import subprocess
+
+from avocado import VERSION
+from avocado import Test
+
+class MinishiftISOTest(Test):
+
+    def setUp(self):
+        ''' Test Setup '''
+        self.test_DIR = os.path.dirname(os.path.realpath(__file__))
+        self.log.info("################################################################")
+        self.log.info("Avocado version : %s" % VERSION)
+        self.log.info("################################################################")
+
+    def test_minishift_centos_iso_build(self):
+        ''' minishift centos iso build test '''
+        self.log.info("Testing minishift centos iso build ...")
+        process = subprocess.Popen("make", shell=True, stdout=subprocess.PIPE)
```

Comment:
> hmm, I think you can assume that the ISO is built when you test.

If we assume so, what to test then? How we will get the ISO?

Also, this is just the start of the test. 
There are other pieces remaining like
- ISO boots properly or not through docker-machine/minishift 
- Call docker-machine/minishift commands to verify things working like docker-machine env etc (whatever possible)